### ISSUE TITLE
Cannot invoke "com.intellij.notification.NotificationGroup.createNotification"

### ISSUE THREAD

**Issue by hdsuperman** [aws/aws-toolkit-jetbrains#3433]
**Your Environment**
- OS: 
- JetBrains product: WebStorm 2022.3.1
- JetBrains product version:  Build #WS-223.8214.51, built on December 20, 2022
- AWS Toolkit version: 1.57-223
- JVM/Python version: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
- Runtime version: 17.0.5+1-b653.23 aarch64

**Additional context**
error logs

```
java.lang.NullPointerException: Cannot invoke "com.intellij.notification.NotificationGroup.createNotification(String, String, com.intellij.notification.NotificationType, com.intellij.notification.NotificationListener)" because "group" is null
	at software.aws.toolkits.jetbrains.core.AwsTelemetryPrompter.runActivity(AwsTelemetryPrompter.kt:22)
	at com.intellij.ide.startup.impl.StartupManagerImpl.runActivityAndMeasureDuration(StartupManagerImpl.kt:340)
	at com.intellij.ide.startup.impl.StartupManagerImpl.access$runActivityAndMeasureDuration(StartupManagerImpl.kt:66)
	at com.intellij.ide.startup.impl.StartupManagerImpl$Companion$addActivityEpListener$1$extensionAdded$1.invokeSuspend(StartupManagerImpl.kt:86)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:570)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:677)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:664)
```

**Comment by am312** [comment]
I'm seeing this when installing AWS Toolkit 1.58-223 using IntelliJ IDEA 2022.3.1 CE.

**Comment by rli** [comment]
Released as part of 1.61
